termprogressbar/progressbar_test.go

```diff
@@ -0,0 +1,65 @@
+package termuiprogressbar_test
+
+import (
+	"bytes"
+	"io"
+	"os"
+	"testing"
+
+	"github.com/hpcloud/termui/termprogressbar"
+	"github.com/stretchr/testify/assert"
+)
+
+// tests if the progressbar is visible if it is initialized with visible=true
+func TestVisible(t *testing.T) {
+	assert := assert.New(t)
+
+	old := os.Stdout
+	r, w, _ := os.Pipe()
+	os.Stdout = w
```

Comment:
Consider doing `defer func() { os.Stdout = old }()` to ensure we never end up not restoring it?